src/protocol/vm/state.rs

```diff
@@ -43,36 +43,36 @@ where
     <D as EngineDatabaseInterface>::Error: std::fmt::Debug,
 {
     /// The pool's identifier
-    pub id: String,
+    id: String,
     /// The pool's token's addresses
     pub tokens: Vec<H160>,
     /// The current block, will be used to set vm context
-    pub block: BlockHeader,
+    block: BlockHeader,
     /// The pools token balances
-    pub balances: HashMap<H160, U256>,
+    balances: HashMap<H160, U256>,
     /// The contract address for where protocol balances are stored (i.e. a vault contract).
     /// If given, balances will be overwritten here instead of on the pool contract during
     /// simulations
-    pub balance_owner: Option<H160>,
+    balance_owner: Option<H160>,
```

Comment:
Question: why not make this pub(super)? Then it can be accessed within this `vm` module as needed, but is considered private outside of the module.

Reply:
hehe didn't know it existed. but is it necessary since it's really only ever accessed for tests?